api/src/form_schema/forms/sf424.py

```diff
@@ -667,231 +667,139 @@
 FORM_UI_SCHEMA = [
     {
         "type": "section",
-        "name": "submission_info",
-        "label": "Submission Type",
-        "children": [
-            {
-                "type": "field",
-                "definition": "/properties/submission_type",
-            }
-        ],
+        "name": "submission_type",
+        "label": "1. Type of Submission",
+        "children": [{"type": "field", "definition": "/properties/submission_type"}],
```

Comment:
should we change the label on the field to match the name of the section? currently "Type of submission" => "submission type" looks weird to me. See also "Type of application" => "application type"